tests/conformance/pubsub/pubsub.go

```diff
@@ -6,35 +6,42 @@
 package pubsub
 
 import (
+	"fmt"
 	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	"github.com/dapr/components-contrib/pubsub"
-	"github.com/dapr/components-contrib/tests/conformance/utils"
+	"github.com/google/uuid"
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 	"k8s.io/apimachinery/pkg/util/sets"
+
+	"github.com/dapr/components-contrib/pubsub"
+	"github.com/dapr/components-contrib/tests/conformance/utils"
 )
 
 const (
-	defaultPubsubName            = "pubusub"
-	defaultTopicName             = "testTopic"
-	defaultMessageCount          = 10
-	defaultMaxReadDuration       = 20 * time.Second
-	defaultWaitDurationToPublish = 5 * time.Second
+	defaultPubsubName             = "pubusub"
+	defaultTopicName              = "testTopic"
+	defaultMessageCount           = 10
+	defaultMaxReadDuration        = 60 * time.Second
+	defaultWaitDurationToPublish  = 5 * time.Second
+	defaultCheckInOrderProcessing = true
+	defaultSimulateErrors         = true
```

Comment:
Why simulate errors is configurable?

Reply:
See above. I'm putting a comment here to consider removing later.